docs/courses/module.md

```diff
@@ -0,0 +1,90 @@
+# LibreLingo Documentation on Module
+
+A **Module** resides below the courses within the course structure. It contains [skills](skill.md).
+
+They have less complexity as they're here just to group skills and nothing else. On the frontend they appear as headings to group skills.
+
+To get a better understanding, you can read the [course basics](README.md#basics).
+
+---
+
+**Confused?**
+[Ask people](https://github.com/kantord/LibreLingo/discussions) on GitHub Discussions.
+
+---
+
+**Table of Contents:**
+- [Tree structure](#tree-structure)
+- [`module.yaml`](#yaml)
+  - [Data breakdown](#data-breakdown)
+- [Creating new module](#creating-new)
+
+## Tree structure
+
+A typical tree structure for a module is like this:
+
+```
+courses/spanish-from-english/basics/
+├── module.yaml
+└── skills
+    ├── animals.yaml
+    ├── clothes.yaml
+    ├── food.yaml
+    ├── nature.yaml
+    ├── plurals.yaml
+    ├── professions.yaml
+    ├── verb_plurals.yaml
+    └── verbs.yaml
+```
+
+Here, `animals.yaml`, `clothes.yaml` and `food.yaml` etc. are [skills](skill.md). Directly inside the module directory there is also a file called `module.yaml`. This file contains information about the module (see below).
+
+NOTE: A module directory name should not have spaces and should be written in `slug-form` in plain English. The human friendly `Name` in the yaml however, can contain uppercase and spaces. The yaml skill files under it should be always inside a directory named `skills`.
```

Comment:
> in plain English

not that the source language doesn't have to be English. It could be any other language

Reply:
That's a good technical question. I think you are right in the sense that it's better to stick to using only the English alphabet, although it doesn't have to be in English, it could just be a transcribed version

Reply:
Probably the main purpose of naming these files is so that the course editors will be able to navigate easily. How it will look in the URL is secondary to that, because most users will navigate through the GUI and not care how the URL looks

Reply:
I see. How about this...?

> NOTE: A module directory name should not have spaces, should be written in `slug-form` and ideally in plain English characters. If your source language is not English, you can try roughly to follow the pronunciation and write it using English characters. ...

Reply:
I've also changed:
> The human friendly `Name` in the yaml however, can contain uppercase and spaces.

to

> The human friendly `Name` in the yaml however, can contain uppercase, spaces and the actual name in source language.

in order to match with the change.